app/views/shared/_event-modal.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @event %>
```

Comment:
I think we should keep the _event_modal.html.erb (notice also the naming convention! snake_case yay!) inside `app/views/home/` as it's only used there, hence not shared!